packages/flutter_tools/lib/src/usage.dart

```diff
@@ -29,6 +29,9 @@ class Usage {
     // default as custom dimension 1 (configured in our analytics account).
     _analytics.setSessionValue('dimension1', os.name);
 
+    // Report the branch (or, "channel") name.
```

Comment:
I'd remove this comment (it doesn't really add anything you can't figure out from the code), remove the blank line before it, and then update the earlier comment to cover both cases.